templates/zerver/help/import-your-settings.md

```diff
@@ -0,0 +1,38 @@
+# Import your settings
+
+When you create a Zulip account using an email address already associated with
+an account in another Zulip organization, you can import your user settings
+from an existing account. It's a convenient way to preserve the user settings
+that you've already customized.
+
+!!! tip ""
+    Settings that may not apply to all organizations, such as custom profile
+    fields, will not be imported.
+
+The import will include your:
+
+- [Name and avatar]({{ root_domain_uri }}/#settings/profile)
+- [Privacy settings]({{ root_domain_uri }}/#settings/account-and-privacy)
+- [Display settings]({{ root_domain_uri }}/#settings/display-settings)
+- [Notification settings]({{ root_domain_uri }}/#settings/notifications)
```

Comment:
Do we need `root_domain_uri` here? I'd expect relative links to be better.

Reply:
It would be ideal if we had something similar to `{relative|gear|settings}` that we could use instead of `root_domain_uri`.

But maybe we should just remove these links and add a link to **Review your settings**:

<img width="348" alt="image" src="https://user-images.githubusercontent.com/2343554/189466051-5a008d63-c561-44a2-8aac-d27261b27621.png">

We could also use `{relative|gear|settings}` in the step-by-step instructions, but I think it would be best to simply link to the **Review your settings** page.

<img width="543" alt="image" src="https://user-images.githubusercontent.com/2343554/189465971-9b6ecbd8-03c8-4ad8-9b81-b3005efadb06.png">

Reply:
Just to be clear, my proposal was that we just write a link to `/#settings/notifications` -- such relative links should Just Work.

Reply:
Cool, you are right. I didn't expect such relative links to just work.